open-craft/openedx-course-enrollment-audit
# feat: initial implementation

This is the initial implementation of a plugin that creates a new DB table and aggregates the `ManualEnrollment` entries with the additional parsing for the `reason` field.

## Testing instructions
1. Use the Redwood devstack/Tutor to test this.
2. Use the instructions from the `Developing -> Tutor Installation` section from the README.
3. Backfill the enrollments with `./manage.py lms backfill_course_enrollment_audit`.
4. Optionally, create some enrollments from the Instructor Dashboard.
5. Check the entries with the Django shell. Example: `from pprint import pprint; from openedx_course_enrollment_audit.models import CourseEnrollmentAudit; [pprint(vars(obj)) for obj in CourseEnrollmentAudit.objects.all()]`

_Private-ref: [BB-9131](https://tasks.opencraft.com/browse/BB-9131)_
